app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2><%= I18n.t('auth.sign_in') %></h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
It would be best to review all your PRs to fix already-raised issues such as https://github.com/Goose97/google-scraper-ruby/pull/55#discussion_r1322658636